docs/associations.md

```diff
@@ -522,6 +522,13 @@ User.findAll({
   }]
 });
 ```
+Is possible rewrite the unique constraint name using **uniqueKey** option.
```

Comment:
Belongs-To-Many creates a unique key when primary key is not present on `through` model. This unique key name can be overridden using **uniqueKey** option.